libs/layer-composer/src/generators/user-points/user-points.ts

```diff
@@ -51,17 +51,17 @@ class UserPointsGenerator {
       'source-layer': DEFAULT_CONTEXT_SOURCE_LAYER,
     }
     let filters: FilterSpecification | undefined
-    if (config?.pointTimeFilterProperty) {
+    if (config?.startTimeFilterProperty && config?.endTimeFilterProperty) {
```

Comment:
Does this keep filtering functionality by a timestamp instead of a range?

Reply:
When we filter by `timestamp` what we are doing is setting the start and end time to be the same number, so yeah, it works both for `timestamp` and `timerange`